src/middlewares.js

```diff
@@ -213,6 +213,7 @@ export async function handleParseHeaders(req, res, next) {
     });
     return;
   }
+  await config.loadKeys();
```

Comment:
@Moumouls do you think this could become a performance issue to load keys in middleware?